src/Raven.Client/Documents/Operations/OngoingTasks/GetOngoingTaskInfoOperation.cs

```diff
@@ -7,12 +7,23 @@
 
 namespace Raven.Client.Documents.Operations.OngoingTasks
 {
+    /// <summary>
+    /// Operation to retrieve detailed information about a specific ongoing task.
+    /// Ongoing tasks include various types of tasks such as replication, ETL, backup, and subscriptions.
+    /// </summary>
     public sealed class GetOngoingTaskInfoOperation : IMaintenanceOperation<OngoingTask>
     {
         private readonly string _taskName;
         private readonly long _taskId;
         private readonly OngoingTaskType _type;
 
+        /// <inheritdoc cref="GetOngoingTaskInfoOperation"/>
+        /// <param name="taskId">The unique identifier of the ongoing task to retrieve information for.</param>
+        /// <param name="type">The type of the ongoing task, such as replication, ETL, or backup.</param>
+        /// <exception cref="ArgumentException">
+        /// Thrown if the specified task type is <see cref="OngoingTaskType.PullReplicationAsHub"/>, which is not supported.
```

Comment:
need to document the `OngoingTaskType` class as well

Reply:
I'm not sure how to document it since this is an `enum` type, and I believe the names of its members are already quite clear